src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -117,6 +120,14 @@ message InterfaceConfigProto {
   repeated RetryParamsDefinitionProto retry_params_def = 31;
 }
 
+message SmokeTestConfigProto {
```

Comment:
It is possible to have auto-generated checks but I would consider it as a future enhancement. Right now the return object will be recursively printed as JSON strings to stdout so the nested values can be checked manually.